### ISSUE TITLE
Consider using NPW for analysis

### ISSUE THREAD

**Issue by dabreegster** [nptscot/corenet#25]
I noticed recent commit e4e50c788f82ff1354a29902890da41e8e822e9f calculates things like population coverage and other performance metrics about the CN. If any of these metrics are useful to the end-user, we could consider adding them to NPW as well. And if it's helpful, it's possible to use NPW to calculate some of these metrics today, by

1) Rebuilding an area with an updated CN gpkg
2) Importing everything from the CN automatically (now that the map-matching is pretty good, this is feasible)
3) Using the existing web app to explore metrics

If there are gaps in step 3 that'd also be helpful for users, or different ways we're calculating things like pop coverage, it could be a good opportunity to share and consolidate methodology?

**Comment by Robinlovelace** [comment]
Sounds good but there would need to be a way to call the NPW code or data from the codebase for this to work. If there's a command line interface we could do something with system calls as we do with `odjitter` but doesn't sound like a priority to me and not even sure if it's worth doing at present.

**Comment by dabreegster** [comment]
True. Fair points, just an idea -- please disregard / don't prioritize
